apps/gp2-frontend/src/NotificationMessages.tsx

```diff
@@ -0,0 +1,29 @@
+import { NotificationContext, Notification } from '@asap-hub/react-context';
+import { FC, useCallback, useState } from 'react';
+
+const NotificationMessages: FC = ({ children }) => {
+  const [notifications, setNotifications] = useState<Notification[]>([]);
+  const addNotification = useCallback(
+    (notification: Notification) => {
+      setNotifications((prev) => [...prev, notification]);
+    },
+    [setNotifications],
+  );
+  const removeNotification = useCallback(
+    (notification: Notification) => {
+      setNotifications((prev) =>
+        prev.filter((n) => n.message !== notification.message),
```

Comment:
Should we check the page here too? For example the same message is displayed on the dashboard and outputs.

Reply:
that's a valid point, I can create a function to validate the notifications